src/sentry/models/integrations/sentry_app.py

```diff
@@ -75,15 +74,6 @@ def default_uuid():
     return str(uuid.uuid4())
 
 
-def generate_slug(name: str, is_internal=False) -> str:
-    slug = slugify(name)
-    # for internal, add some uuid to make it unique
-    if is_internal:
-        slug = f"{slug}-{default_uuid()[:UUID_CHARS_IN_SLUG]}"
-
-    return slug
```

Comment:
Moved to `_generate_and_validate_slug` [here](https://github.com/getsentry/sentry/pull/59643/files#diff-b99cd408c8c56c64189f36a38ca12097a8fe4b8634de90696fa5b76d0925f6fbR299-R303) b/c I was running into and import error. Also the function is only called there, and the func name `_generate_and_validate_slug` already encapsulates the functionality